pkg/scheduler/util.go

```diff
@@ -76,7 +77,7 @@ func unmarshalSchedulerConf(confStr string) ([]framework.Action, []conf.Tier, []
 		if action, found := framework.GetAction(strings.TrimSpace(actionName)); found {
 			actions = append(actions, action)
 		} else {
-			return nil, nil, nil, fmt.Errorf("failed to found Action %s, ignore it", actionName)
+			klog.Errorf("failed to found Action %s, ignore it", actionName)
```

Comment:
If no action configured, scheduler will make no sense for no scheduling process. Enqueue and Allocate is necessary.

Reply:
There is an action's array loop outside the code. So if only one action config is not found, should all configurations be discarded?

Reply:
Yes, i think it is reasonable. Even if one invalid action configuration is not allowed.

Reply:
@Thor-wl Thanks for the explanation. Maybe the error message need to be more accurate.